src/types/response/futures.ts

```diff
@@ -146,34 +152,6 @@ export interface FuturesAccountBookRecord {
   id: string;
 }
 
-export interface UpdateFuturesDualModeResp {
```

Comment:
Instead of deleting (breaking change if anyone is using it), mark as deprecated and point to the alternative please